tests/components/ring/test_init.py

```diff
@@ -119,26 +129,26 @@ async def test_auth_failure_on_global_update(
     await hass.config_entries.async_setup(mock_config_entry.entry_id)
     await hass.async_block_till_done()
     assert not any(mock_config_entry.async_get_active_flows(hass, {SOURCE_REAUTH}))
-    with patch(
-        "ring_doorbell.Ring.update_devices",
-        side_effect=AuthenticationError,
-    ):
-        async_fire_time_changed(hass, dt_util.now() + timedelta(minutes=20))
-        await hass.async_block_till_done()
 
-        assert "Authentication failed while fetching devices data: " in [
-            record.message
-            for record in caplog.records
-            if record.levelname == "ERROR"
-            and record.name == "homeassistant.components.ring.coordinator"
-        ]
+    mock_ring_client.update_devices.side_effect = AuthenticationError
+
+    async_fire_time_changed(hass, dt_util.now() + timedelta(minutes=20))
+    await hass.async_block_till_done()
+
+    assert "Authentication failed while fetching devices data: " in [
+        record.message
+        for record in caplog.records
+        if record.levelname == "ERROR"
+        and record.name == "homeassistant.components.ring.coordinator"
+    ]
```

Comment:
For the future, it would be nice if we instead could measure an update failure with the effects on the config entry or the entities.